lib/scripts/windows/get_tags.ps1

```diff
@@ -0,0 +1,42 @@
+$vcapApp = $env:VCAP_APPLICATION
+$vcapApp = $vcapApp | ConvertFrom-JSON
+
+$vcap_variables = "application_id", "name", "instance_index", "space_name"
```

Comment:
Could we have some comments to understand what this does and how it works?

Reply:
Good call, I'll add some comments. Its mostly just a port of the python script we're using for linux - https://github.com/DataDog/datadog-cloudfoundry-buildpack/blob/master/lib/scripts/get_tags.py

Reply:
I think you skipped this, didn't you?

Reply:
I had added some comments, but I just added another one at the top of the file. Let me know if you think more is needed here.